Description: changing the name of the JNI library shipped in libjogl2-jni,
 which changed in libjogl2-java/2.4.0+dfsg.
Author: Pierre Gruet <pgt@debian.org>
Forwarded: not-needed
Last-Update: 2023-04-22

--- a/scilab/configure.ac
+++ b/scilab/configure.ac
@@ -1065,8 +1065,8 @@
                    LDFLAGS="$LDFLAGS -L/usr/lib/gluegen2 -L/usr/lib64/gluegen2"         # RedHat
                    LDFLAGS="$LDFLAGS -L$SCI_SRCDIR/thirdparty -L$SCI_SRCDIR/lib/thirdparty -L$SCI_SRCDIR/bin"        # Scilab thirdparties
                    symbol="Java_jogamp_common_jvm_JVMUtil_initialize"
-                   AC_CHECK_LIB([gluegen_rt], [$symbol], [GLUEGEN2_RT_LIBS="-lgluegen_rt"],
-                   [AC_MSG_ERROR([libgluegen_rt: Library missing (Cannot find symbol $symbol). Check if libgluegen_rt - C/Java (JNI) interface for GLUEGEN2 - is installed and if the version is correct. Note that you might have to update etc/librarypath.xml to provide the actual path to the JNI libraries.])],
+                   AC_CHECK_LIB([gluegen2_rt], [$symbol], [GLUEGEN2_RT_LIBS="-lgluegen2_rt"],
+                   [AC_MSG_ERROR([libgluegen2_rt: Library missing (Cannot find symbol $symbol). Check if libgluegen_rt - C/Java (JNI) interface for GLUEGEN2 - is installed and if the version is correct. Note that you might have to update etc/librarypath.xml to provide the actual path to the JNI libraries.])],
                    [-ldl])
                    LDFLAGS=$LDFLAGS_save
                 fi
--- a/scilab/modules/gui/src/java/org/scilab/modules/gui/SwingView.java
+++ b/scilab/modules/gui/src/java/org/scilab/modules/gui/SwingView.java
@@ -161,7 +161,7 @@
 
     static {
         try {
-            System.loadLibrary("gluegen_rt");
+            System.loadLibrary("gluegen2_rt");
         } catch (Exception e) {
             System.err.println(e);
         }
